components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/design-grid.js

```diff
@@ -978,14 +983,16 @@ define(['require', 'log', 'jquery', 'backbone', 'lodash', 'designViewUtils', 'dr
                     function popOverForConnector() {
                         $('#' + connectionObject.id).popover({
                             trigger: 'focus',
-                            title: 'Are you sure you want to delete?',
+                            title: 'Confirmation',
                             html: true,
                             content: function () {
                                 return $('.pop-over').html();
 
                             }
                         });
+                        $('#' + connectionObject.id).off();
                         $('#' + connectionObject.id).popover("show");
+                        $('.no').focus();
```

Comment:
please use a better class name than "no"

Reply:
Added the class names for yes and no buttons as 'btn_yes' and 'btn_no' respectively